src/line-graph.js

```diff
@@ -276,10 +277,15 @@ export class LineGraph extends HTMLElement {
    * @param {data} data
    */
   setLegend(svg, height, data) {
-    var j = -1;
```

Comment:
ohh. i know i have started this work, and these variable is used for making columns, if i am not wrong, so it would be better if we can add some description for these variable now.

Reply:
sure!